state/execution.go

```diff
@@ -178,7 +178,7 @@ func (blockExec *BlockExecutor) ProcessProposal(
 		Time:               block.Header.Time,
 		Txs:                block.Data.Txs.ToSliceOfBytes(),
 		SquareSize:         block.Data.SquareSize,
-		DataRootHash:       block.Data.GetDataRootHash(),
+		DataRootHash:       block.Data.Hash(),
```

Comment:
This is effectively the same here, right?

Only catch is if somehow the `hash` is nil, we end up with merkle root of txs instead of a data square root.

```go
func (data *Data) Hash() cmtbytes.HexBytes {
	if data == nil {
		return (Txs{}).Hash()
	}
	if data.hash == nil {
		data.hash = data.Txs.Hash() // NOTE: leaves of merkle tree are TxIDs
	}
	return data.hash
}

```

Reply:
Why did this change in this PR? Seems unrelated to the timeouts

Reply:
minor cleanup, i accidentally pushed to this pr